### DISCUSSION TITLE
Number Phone Masking

### DISCUSSION THREAD

**Discussion by 0xspade** [simple-login/app#841] [Q&A] [answered]
I just saw your tweet regarding an upcoming service phone number masking, My question is, Is it possible to self-host that service? Another question is, Is it going to available to all countries or it's just a limited for certain countries?

**Comment by nguyenkims** [comment, marked as answer]
This service can be self hosted and should work for most countries.

**Reply by 0xspade** [reply]
Last question, Is it possible to send and receive from those phone numbers that are generated by the upcoming phone number masking service?

**Reply by nguyenkims** [reply]
Our current plan is to only allow receiving messages. We'll might add the possibility to send messages in the future.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@nguyenkims Q: in addition to doing this, is it possible for u guys to provide a privacy first data plan or something in the future? Is it something ur looking into or is this out of ur scope
